docs/conf.py

```diff
@@ -42,12 +42,14 @@
 # extensions coming with Sphinx (named 'sphinx.ext.*') or your custom
 # ones.
 extensions = [
+    "plantweb.directive",
     "sphinx.ext.autodoc",
     "sphinx.ext.coverage",
     "sphinx.ext.doctest",
     "sphinx.ext.intersphinx",
     "sphinx.ext.napoleon",
     "sphinx.ext.viewcode",
+    "sphinxcontrib.mermaid",
```

Comment:
aren't actually using mermaid anywhere, right?

Reply:
not yet but since I added it anyways and we can do more mermaid stuff in the future, I saw no reason to remove this.

Reply:
there's also no reason to keep it here for some hypothetical future :)

Reply:
I could just open an issue "Add mermaid support for docs" and change this to "closes #thatissue" and then it's a feature! :stuck_out_tongue_closed_eyes: